icaruscode/Decode/fcl/stage0_run1_rawpmt_icarus.fcl

```diff
@@ -0,0 +1,25 @@
+###
+## This fhicl file is used to run "stage0" processing specifically for the case where all
+## TPC data is included in an artdaq data product from a single instance
```

Comment:
I don't understand this comment. Is this outdated by now?
```suggestion
##
## Standard Stage0 processing, but it explicitly preserves optical waveforms.
##
```

Reply:
yes, I believe so. I do not know who the original author of this fhicl file is/was... I imagine they copied and modified. So, yes, should be modernized.